Makefile

```diff
@@ -55,6 +55,13 @@ e2e-aci: ## Run End to end ACI tests. Set E2E_TEST=TestName to run a single test
 e2e-ecs: ## Run End to end ECS tests. Set E2E_TEST=TestName to run a single test
 	go test -timeout 20m -count=1 -v $(TEST_FLAGS) ./tests/ecs-e2e
 
+cross-%:
```

Comment:
nit: maybe define some targets with explicit os/arch delegating to this one, so we know which ones are actually supported, and we have auto-completion on targets

Reply:
of course we would need different names for these explicit targets, so we can delegate to the generic one, so maybe only `cross-mac`, `cross-win`, `cross-arm` ? 
If this is not an easy fix, forget about it, just a small UX improvement

Reply:
We could add something like that as a wrapper on top. `cross-mac` should likely do `darwin/amd64` and `darwin/arm64` (i.e.: `cross-mac: cross-darwin-amd64 cross-darwin-arm64`) for example

Not sure what `cross-arm` would do?